ai/smart-labeling/concepts.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Smart Labeling Concepts
+description: This page explains all the concepts related to Smart Labeling
+tags: smart labeling concepts
+---
+
+<Concept opened>
+
+## Admin credentials
+
+The login credentials of the Admin account for the annotation server. 
+The Admin user has access to all the data within the Labeling Workspace and all the functionalities of the annotation server. 
+The Admin can also set permissions for other users of the annotation server.
+
+</Concept>
+
+<Concept opened>
+
+## Annotator
+
+An Annotator is any user who has created an account on the annotation server, other than the Admin.
+
+</Concept>
+
+<Concept opened>
+
+## Annotation server
+
+A public server, connected to the object storage buckets associated with the Labeling Workspace, that is running annotation software.
```

Comment:
```suggestion
A public server, connected to the object storage buckets associated with the labeling workspace, that is running annotation software. It can be used to annotate image and video files in the buckets.
```